src/main/java/de/qabel/desktop/ui/accounting/menuqr/MenuQRController.java

```diff
@@ -0,0 +1,68 @@
+package de.qabel.desktop.ui.accounting.menuqr;
+
+import de.qabel.core.config.Identity;
+import de.qabel.desktop.ui.AbstractController;
+import de.qabel.desktop.ui.accounting.qrcode.QRCodeController;
+import de.qabel.desktop.ui.accounting.qrcode.QRCodeView;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Pane;
+import javafx.scene.layout.VBox;
+import org.controlsfx.control.PopOver;
+
+import javax.inject.Inject;
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class MenuQRController extends AbstractController implements Initializable {
+
+    @Inject
+    private Pane layoutWindow;
+
+    @FXML
+    private AnchorPane menuQR;
+
+    @FXML
+    private VBox vboxMenu;
+
+    private QRCodeView qrcodeView;
+    private QRCodeController qrcodeController;
+    private PopOver popOver;
+
+    @Override
+    public void initialize(URL location, ResourceBundle resources) {
+        initializeQRPopup();
+        initializePopOver();
```

Comment:
If you got 20 menu items, you would always create (and add) 20 new Views. But you never need more than one...

Delay the view initialization to the related `open` method